lib/transforms/reviewSubResourceIntegrity.js

```diff
@@ -0,0 +1,53 @@
+var crypto = require('crypto');
+
+module.exports = function (queryObj, options) {
+    options = options || {};
+    return function reviewSubResourceIntegrity(assetGraph) {
+        assetGraph.findAssets(queryObj || { type: 'Html', isInline: false, isFragment: false, isLoaded: true }).forEach(function (htmlAsset) {
+            var changed = false;
+            assetGraph.findRelations({from: htmlAsset, type: ['HtmlStyle', 'HtmlScript'], to: {isInline: false, isLoaded: true}}).forEach(function (relation) {
```

Comment:
Yes, I agree that it would be very useful to get a warning when an `integrity` attribute that has been manually added to the development build no longer matches. The drawback would be build performance, of course, suggesting that we add a separate tool for it. Also, spinning up the project in the development setting would immediately reveal this situation as the `integrity` attribute would be verified by the browser and the resource would be blocked.